testing/scenario_app/README.md

```diff
@@ -0,0 +1,42 @@
+# Scenario App
+
+This folder contains a dart:ui application and scripts to compile it to AOT
+for exercising embedders.
+
+It intentionally has no dependencies on the Flutter framework or tooling, such
+that it should be buildable as a presubmit or postsubmit to the engine even in
+the face of changes to Dart or dart:ui that require upstream changes in the
+Flutter tooling.
+
+To add a new scenario, create a new subclass of `Scenario` and add it to the
```

Comment:
nit: feel free to submit as is and I can move stuff around later. 

But I think we'll probably want a directory structure like

```
testing/integration
  a bunch of tooling files
  dart/
    some base dart:ui interfaces
    simple/
      *.dart
    channel
      *.dart
  iOS/
    startup/
      code/
      test/
    memory/
      code/
      test/
    scenario_1/
      code/
      test/
    etc/...
```

i.e. I think there will be a lot of platform folders and not that many dart folders